src/mugs.js

```diff
@@ -719,6 +719,15 @@ define([
             data.instances = _.extend(data.instances || {},
                                       mug.form.parseInstanceRefs(value));
         }
+        try {
+            if (value) {
+                value = mug.form.xpath.parse(value.toString()).toHashtag();
+            }
+        } catch (err) {
+            if (_.isString(value) && !value.startsWith('#invalid/')) {
+                value = '#invalid/xpath ' + value;
```

Comment:
The copy-paste tests should be testing this I believe https://github.com/dimagi/Vellum/pull/616/commits/404506461f7f5e246b5c707b19fd7b1d8efd69bc https://github.com/dimagi/Vellum/pull/616/commits/1f4c067fcd82a9acbe99c293cbc3f93288fe0e2a